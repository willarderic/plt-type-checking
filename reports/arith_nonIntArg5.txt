TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: DIV  LEXEME: /						TOKEN: DIV       LEXEME: /
ARITH_OP -> /							ARITH_OP -> /
BIN_OP -> ARITH_OP						BIN_OP -> ARITH_OP
TOKEN: T  LEXEME: t						TOKEN: T         LEXEME: t
CONST -> t							CONST -> t
EXPR -> CONST							EXPR -> CONST
TOKEN: NIL  LEXEME: nil						TOKEN: NIL       LEXEME: nil
CONST -> nil							CONST -> nil
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 1 must be integer					Line 1: Arg 1 must be integer
