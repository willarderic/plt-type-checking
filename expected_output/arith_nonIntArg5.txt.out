TOKEN: LPAREN    LEXEME: (
TOKEN: DIV       LEXEME: /
ARITH_OP -> /
BIN_OP -> ARITH_OP
TOKEN: T         LEXEME: t
CONST -> t
EXPR -> CONST
TOKEN: NIL       LEXEME: nil
CONST -> nil
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 1 must be integer
